src/api/app/models/binary_release.rb

```diff
@@ -87,7 +87,15 @@ def self.update_binary_releases_via_json(repository, json, time = Time.now)
           hash[:binary_updateinfo] = binary['updateinfoid']
           hash[:binary_updateinfo_version] = binary['updateinfoversion']
         end
-        source_package = Package.striping_multibuild_suffix(binary['package'])
+
+        # This is a hack to attempt to get a deeper understanding when there is no binary['package']
+        source_package = if binary['package']
+                           Package.striping_multibuild_suffix(binary['package'])
+                         else
+                           Airbrake.notify('Binary release with no package payload', binary: binary)
+                           nil
+                         end
```

Comment:
```suggestion
        # Send the binary to Airbrake, to get a deeper understanding when there is no binary['package']
        Airbrake.notify('Binary release with no package payload', binary: binary) unless binary['package']
        source_package = Package.striping_multibuild_suffix(binary['package'])
```
Maybe this way it is clearer that there is no change involved in the package tracking history. This would also get rid of the Brakeman warning.

Reply:
Okay, let's try it!